contributing_to_docs/contributing.adoc

```diff
@@ -0,0 +1,143 @@
+= Contribute to OpenShift documentation
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+:description: Basic information about the OpenShift GitHub repository
+
+toc::[]
+
+== Different ways to contribute
+There are a few different ways you can contribute to OpenShift documentation:
+
+* Submit comments at the bottom of each topic (still awaiting implementation)
+* Email the OpenShift documentation team openshift-docs@redhat.com
+* https://github.com/openshift/openshift-docs/issues/new[Create an issue in GitHub] 
+* If you would like to do the work yourself, or if it is a substantial, then you you should clone the repository, make your changes, and submit a PR
+
+*What happens when you submit a PR?*
+
+If you make a substantial change and submit a PR in GitHub, your PR will be reviewed by our documentation team. If there are any further changes, updates, or corrections required we will inform you in the PR. When the PR has been reviewed and all changes have been accepted by our documentation team, you PR will be merged.
```

Comment:
Why "substantial"? I would welcome minor changes/fixes, too.

Also, can you set your editor to hard-wrap your lines at 80? Tim, Brice, and myself have been doing so. So that the source text is easily readable from wherever. E.g., see how this scrolls way off:

https://raw.githubusercontent.com/CowboysFan/openshift-docs/readme_process_updates/contributing_to_docs/contributing.adoc

Makes it easier to comment on individual lines in a PR, too. I would recommend adding something about line length in the guidelines, as well (been meaning to mention that).

At the end of this paragraph:

s/you PR/your PR/